src/content/docs/opentelemetry/best-practices/opentelemetry-manage-ingest-volume.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Manage OpenTelemetry Ingest Volume
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - OTLP
+metaDescription: Manage OpenTelemetry Ingest Volume
+freshnessValidatedDate: never
+redirects:
+---
+
+One of OpenTelemetry's core strengths is its rich set of tools providing unparalleled control over your telemetry data pipeline. This control complements New Relic's [consumption-based pricing](https://newrelic.com/pricing#pricing_plan-data) model.
+
+This page talks about a variety of concepts that contribute to data volume when using OpenTelemetry with New Relic, and tools / patterns available to manage your telemetry data pipeline. It's organized into sections talking about key concepts contributing to data volume for [resource](#concept-resources), [traces](#concept-traces), [metrics](#concept-metrics), and [logs](#concept-logs), followed by a [catalog of tools](#tool-catalog) available for helping manage volume.
+
+## Key concepts: Resources [#concept-resources]
+
+[OTLP](https://opentelemetry.io/docs/specs/otlp/) (OpenTeLemetry Protocol) defines a similar hierarchical message structure across all signals, which avoid repetition at the protocol level by sharing information across records.
+
+* Each export request contains many `Resource{SignalRecord}`s
+* Each `Resource{SignalRecord}` contains many `Scope{SignalRecord}`s
+* Each `Scope{SignalRecord}` contains many `{SignalRecord}`s
+
+Where `{SignalRecord}` is `Span`, `Metric`, and `LogRecord`.
+
+As discussed in the pages discussing ingestion of [Traces](/docs/opentelemetry/best-practices/opentelemetry-best-practices-traces/), [Metrics](/docs/opentelemetry/best-practices/opentelemetry-best-practices-metrics/), and [Logs](/docs/opentelemetry/best-practices/opentelemetry-best-practices-logs/) sent to the [New Relic OTLP Endpoint](/docs/opentelemetry/best-practices/opentelemetry-otlp/), this hierarchical structure is flattened during New Relic and each resource attribute is denormalized onto individual `Span`, `Metric`, and `Log` records.
+
+Most OpenTelemetry language implementations provide packages with [resource detectors](https://opentelemetry.io/docs/specs/otel/resource/sdk/#detecting-resource-information-from-the-environment), which contribute resource attributes based on information detected in the environment. These attributes can be extremely useful, but may include more information than necessary.
+
+See "SDK resource detectors", "Collector resource processor", and "Collector transform processor" in the [tool catalog](#tool-catalog).
```

Comment:
I keep going back and forth on this. Some of the tools in the table are only applicable to a single concept. If this was the case everywhere, it would be a nobrainer to have mini tables in each concept section. But several tools, especially the collector, are referenced over and over again.

> Since we don't have a good way to link to table rows

I'm going to do some experiments to see if we _can_ actually link to table rows. It seems like it should be possible.

Reply:
Linking to rows does work! Checkout the cumulative to delta processor link [here](https://deploy-preview-17915--docs-website-netlify.netlify.app/docs/opentelemetry/best-practices/opentelemetry-manage-data-ingest-volume/#concept-aggregation-temporality) in the latest preview!

Reply:
Ok, pushed a commit updating all of these references to be links to the rows in the tool catalog.